### PR TITLE
Issue 4359 - Tabular view: Group By

### LINKED ISSUE #4359: Tabular view - Collapsible table
### Description
For tabular view we need a table of tickets that presents the information in a sensible manner for the context.
Use the same criteria for showing the extended properties as we do in the ticket list inside the viewer.

### Goals
- [ ] To have a reusable table for the results in tabular view

### Tasks
- [ ] Create a collapsible table with the information that is shown in the  design
- [ ] Add the different  columns depending if the template has enabled the appropiate properties (like risk or issue properties)
- [ ] Try to reuse or adapt the special components like asignees or status level of risk or status, etc

### Notes
This will be a branch from #4347 

### Related Resources
[MockUp](https://www.figma.com/file/bRqIl0g9psqD3ISkmxwcNp/3D-Repo?type=design&node-id=342-37215&mode=design&t=qSsF1vfLNlAAPyFD-0)

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #4359 

#### Description
- Allow to group tickets by properties
- Refine URL search params
- Include "new ticket" in the group section
- Refactor selection page

#### Test cases
Group tickets by different group options

### CODE REVIEW

File: frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableRow/ticketsTableRow.component.tsx [hunk 1]
```diff
@@ -38,22 +49,61 @@ export const TicketsTableRow = ({ ticket, onClick }: { ticket: ITicket, onClick:
...
+					<OverflowContainer>
+						<Highlight search={query}>
+							{title}
+						</Highlight>
+					</OverflowContainer>
+				</Tooltip>
+			</Cell>
+			<Cell>
+				{assignees?.length && (<AssigneesSelect value={assignees} multiple disabled />)}
+				{!assignees?.length && (
```
Comment (The-Daniel) [inline frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableRow/ticketsTableRow.component.tsx:81]: image
If the assignees is an empty array the component is prefaced by a "0". 

Also the ticket above doesn't have an assignees key in the ticket object (hence it doesn't have this problem). It is the same template, so this points to another problem in that I think the property isn't added to the redux state until it is edited.

File: frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.components.tsx
Comment (The-Daniel) [inline frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.components.tsx:1]: filename should be *component* not *components*

File: frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableGroup.component.tsx [hunk 2]
```diff
@@ -16,37 +16,74 @@
...
-import { Headers } from './ticketsTableGroup.styles';
+import AddCircleIcon from '@assets/icons/filled/add_circle-filled.svg';
+import { Header, Headers, Group, NewTicketRow, NewTicketText } from './ticketsTableGroup.styles';
 import { TicketsTableRow } from './ticketsTableRow/ticketsTableRow.component';
+import { NewTicketMenu } from '../../newTicketMenu/newTicketMenu.component';
 
-export const TicketsTableGroup = ({ tickets, onTicketClick }) => {
-	if (!tickets?.length) return (<button type="button"> create new ticket </button>);
-
+type TicketTableGroupProps = {
```
Comment (The-Daniel) [inline frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTableContent/ticketsTableGroup/ticketsTableGroup.component.tsx:26]: Should probably be name TicketsTableGroupProps to match the component

**Review by The-Daniel — CHANGES_REQUESTED** [review]

File: frontend/src/v5/store/tickets/tickets.selectors.ts [hunk 3]
```diff
@@ -39,10 +39,10 @@ export const selectTicketsByContainersAndFederations = createSelector(
 	selectTicketsDomain,
 	(state, containersAndFederationsIds: string[]) => containersAndFederationsIds,
 	(state, containersAndFederationsIds) => {
-		const tickets = containersAndFederationsIds.flatMap((modelId) => (
+		const ticketsWithModelId: Array<ITicket & { modelId: string }> = containersAndFederationsIds.flatMap((modelId) => (
```
[the remark this answers is not in the document]
Comment (Amantini1997) [reply frontend/src/v5/store/tickets/tickets.selectors.ts:42]: Good shout!

**Review by The-Daniel — COMMENTED** [review]
Just that one small change and I think I'm okay to approve

**Pull request merged**